File: implementation/scipamato/src/test/java/ch/difty/scipamato/web/WicketUtilsTest.java
package ch.difty.scipamato.web;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

import ch.difty.scipamato.FinalClassTest;

public class WicketUtilsTest extends FinalClassTest<WicketUtils> {

    @Test
    public void labelTag() {
        assertThat(WicketUtils.LABEL_TAG).isEqualTo("Label");
    }

    @Test
    public void labelResourceTag() {
        assertThat(WicketUtils.LABEL_RECOURCE_TAG).isEqualTo(".label");
    }

    @Test
    public void shortLabelResourceTag() {
        assertThat(WicketUtils.SHORT_LABEL_RECOURCE_TAG).isEqualTo(".short.label");
    }

    @Test
    public void panelHeaderResourceTag() {
        assertThat(WicketUtils.PANEL_HEADER_RESOURCE_TAG).isEqualTo(".header");
    }

}

File: common/common-wicket/src/test/kotlin/ch/difty/scipamato/common/web/WicketUtilsTest.kt
package ch.difty.scipamato.common.web

import org.amshove.kluent.shouldBeEqualTo
import org.junit.jupiter.api.Test

internal class WicketUtilsTest {

    @Test
    fun labelTag() {
        LABEL_TAG shouldBeEqualTo "Label"
    }

    @Test
    fun labelResourceTag() {
        LABEL_RESOURCE_TAG shouldBeEqualTo ".label"
    }

    @Test
    fun shortLabelResourceTag() {
        SHORT_LABEL_RESOURCE_TAG shouldBeEqualTo ".short.label"
    }

    @Test
    fun panelHeaderResourceTag() {
        PANEL_HEADER_RESOURCE_TAG shouldBeEqualTo ".header"
    }
}
